iosApp/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -62,19 +76,19 @@ struct LoginView: View {
     }
 
     var loginField: some View {
-        TextField(Localize.loginFieldsEmail(), text: $email)
+        TextField(Localize.loginFieldEmail(), text: $email)
             .keyboardType(.emailAddress)
             .primaryTextField()
             .accessibility(.login(.emailField))
     }
```

Comment:
I think we should name this View to `emailField` for consistency 🙏 Btw, what do you think about extracting the `loginField` to a new file so it they can be shared between the `Login` and `ForgotPassword` screen?  🤔

Reply:
I think sharing the style through `primaryTextField` is enough for this view?

Reply:
@blyscuit okay sure, just an optional suggestion at this point. We can definitely revisit if the sharing needs is more imminent 🙏